k9mail/src/main/java/com/fsck/k9/Account.java

```diff
@@ -760,7 +760,7 @@ private void resetVisibleLimits() {
         try {
             getLocalStore().resetVisibleLimits(getDisplayCount());
         } catch (MessagingException e) {
-            Log.e(K9.LOG_TAG, "Unable to reset visible limits", e);
+            Timber.e("Unable to reset visible limits", e);
```

Comment:
With Timber exceptions are the first argument. Format string arguments go at the end.